Marlin/src/pins/pins_BIGTREE_SKR_MINI_V1_1.h

```diff
@@ -0,0 +1,193 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (C) 2016 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (C) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef TARGET_STM32F1
+  #error "Oops! Select an STM32F1 board in 'Tools > Board.'"
+#endif
+
+#ifndef BOARD_NAME
+  #define BOARD_NAME "BIGTREE SKR mini V1.1"
+#endif
+
+// Ignore temp readings during develpment.
+//#define BOGUS_TEMPERATURE_FAILSAFE_OVERRIDE
+
+//
+// Limit Switches
+//
+#define U_MIN_PIN          -1
+#define V_MIN_PIN          -1
+#define W_MIN_PIN          -1
+#define X_STOP_PIN         PC2
+#define Y_STOP_PIN         PC1
+#define Z_STOP_PIN         PC0
+
+//
+// Steppers
+//
+
+/**
+ * TODO: Currently using same Enable pin for all steppers.
+ */
+
+#define X_STEP_PIN         PC6
+#define X_DIR_PIN          PC7
+#define X_ENABLE_PIN       PB15
+//#ifndef X_CS_PIN
+//  #define X_CS_PIN         PC10
+//#endif
+
+#define Y_STEP_PIN         PB13
+#define Y_DIR_PIN          PB14
+#define Y_ENABLE_PIN       PB12
+//#ifndef Y_CS_PIN
+//  #define Y_CS_PIN         PC11
+//#endif
+
+#define Z_STEP_PIN         PB10
+#define Z_DIR_PIN          PB11
+#define Z_ENABLE_PIN       PB2
+//#ifndef Z_CS_PIN
+//  #define Z_CS_PIN         PC12
+//#endif
+
+#define E0_STEP_PIN        PC5
+#define E0_DIR_PIN         PB0
+#define E0_ENABLE_PIN      PC4
+//#ifndef E0_CS_PIN
+//  #define E0_CS_PIN        PC13
+//#endif
+
+//#define E1_STEP_PIN        PC8
+//#define E1_DIR_PIN         PC9
+//#define E1_ENABLE_PIN      PA8
+
+//#define E2_STEP_PIN        PC10
+//#define E2_DIR_PIN         PC11
+//#define E2_ENABLE_PIN      PA8
+
+#if ENABLED(TMC_USE_SW_SPI)
+  #define TMC_SW_SCK       PB3
+  #define TMC_SW_MISO      PB4
+  #define TMC_SW_MOSI      PB5
+#endif
+
+//
+// Misc. Functions
+//
+
+//#define LED_PIN            PD2
+
+//
+// Heaters / Fans
+//
+#define HEATER_0_PIN       PA8   // EXTRUDER 1
+//#define HEATER_1_PIN       PD2
```

Comment:
PD2 is X+

Reply:
No ,PA2 is X+ ,If you have a board in your hand, you can see on the back of the board there is a silk screen in the top left corner

Reply:
right, sorry, PD2 is a EXP2 pin, btenc1